### ISSUE TITLE
date2InstantTypeMapper value variable name used at PgDateSupportJoda

### ISSUE THREAD

**Issue by margussipria** [tminglei/slick-pg#370]
This works fine if you start new project, and select just one date support class, there does not have conflict, but if you have old codebase and start converting date columns, then this fails... and you have to add extra code.

Also there are duplicate method names `date2Timestamp1ColumnExtensionMethods`, `date2Timestamp1OptColumnExtensionMethods` and also class `PgDate2TimePositionedResult` name is duplicated.

This is present with new `0.14.8` and `0.15.x`.

**Comment by tminglei** [comment]
Sorry, I did not understand it.

**Comment by margussipria** [comment]
```scala
object MyPostgresDriver extends MyPostgresDriver

trait TestPostgresDriver
  extends ExPostgresDriver
    with PgDateSupportJoda
    with PgDate2Support { driver =>

  override val api = new API
    with JodaDateTimeImplicits
    with Date2DateTimeImplicitsDuration
}
```

will throw:

```
[error] /src/main/scala/utils/MyPostgresDriver.scala:107: overriding value date2InstantTypeMapper in trait JodaDateTimeImplicits of type slick.jdbc.JdbcType[org.joda.time.Instant];
[error]  value date2InstantTypeMapper in trait Date2DateTimeImplicits of type slick.jdbc.JdbcType[java.time.Instant] has incompatible type
[error]   override val api = new API
[error]                ^
[error] /src/main/scala/utils/MyPostgresDriver.scala:107: <$anon: TestPostgresDriver.this.API with TestPostgresDriver.this.JodaDateTimeImplicits with TestPostgresDriver.this.Date2DateTimeImplicitsDuration> inherits conflicting members:
[error]   value date2InstantTypeMapper in trait JodaDateTimeImplicits of type slick.jdbc.JdbcType[org.joda.time.Instant]  and
[error]   value date2InstantTypeMapper in trait Date2DateTimeImplicits of type slick.jdbc.JdbcType[java.time.Instant]
[error] (Note: this can be resolved by declaring an override in <$anon: TestPostgresDriver.this.API with TestPostgresDriver.this.JodaDateTimeImplicits with TestPostgresDriver.this.Date2DateTimeImplicitsDuration>.)
[error]   override val api = new API
```

**Comment by tminglei** [comment]
@margussipria released in slick-pg `v0.14.9`/`v0.15.4`. 👍
